Sources/Sentry/SentryScreenshot.m

```diff
@@ -47,7 +47,12 @@ - (void)saveScreenShots:(NSString *)path
 
         if ([window drawViewHierarchyInRect:window.bounds afterScreenUpdates:false]) {
             UIImage *img = UIGraphicsGetImageFromCurrentImageContext();
-            [result addObject:UIImagePNGRepresentation(img)];
+            if (img.size.width > 0 || img.size.height > 0) {
```

Comment:
`m`: Can we please add tests for this new functionality? As this code is not really testable you could extract this code into a method, expose it via a private category that you would have to add similar as we have it, for example, for SentryClient+Private.h, and then call it from tests. I can also push the basic solution to your branch if you want @krystofwoldrich.

Reply:
You can push the solution to this branch and I will work with that. It will definitely help me.

Besides exposing it with the private header. Are there any examples in the test on how to mock functions, likely I will have to mock `UIGraphicsGetImageFromCurrentImageContext`.

Reply:
The code was already testable. I added a test.